src/main/kotlin/de/achimonline/ansible_lint/command/AnsibleLintCommandLine.kt

```diff
@@ -11,6 +11,7 @@ class AnsibleLintCommandLine(private val settings: AnsibleLintSettings = Ansible
     fun createVersionCheckProcess(workingDirectory: String): Process {
         return GeneralCommandLine()
             .withEnvironment(System.getenv())
+            .withEnvironment("PATH", settings.path)
```

Comment:
This will completely override any existing `PATH` settings of the current system/environment settings.